current/advanced/block-import.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: block-import
+lang: en
+title: The Block Import Pipeline
+---
+
+## The Import Queue
+
+The import queue is an abstract worker queue present in every Substrate node. It is not part of the
+runtime. The Import Queue is responsible for processing pieces of incoming information, verifying
+them, and if they are valid, importing them into the node's state. The most fundamental piece of
+information that the import queue processes is blocks themselves, but it is also responsible for
+importing consensus-related messages such as Justifications and, in light clients, finality proofs.
+
+The import queue collects incoming blocks from the network and stores them in a pool. The blocks are
+later checked for validity and discarded if they are not valid. Blocks that are valid are then
+imported into the node's local state.
+
+The import queue is codified abstractly in Substrate by means of the
+[`ImportQueue` trait](https://substrate.dev/rustdocs/v2.0.0-alpha.8/sp_consensus/import_queue/trait.ImportQueue.html).
```

Comment:
I think we want all links to go to `crates.io`, right? Honestly, even I am starting to lose track :sweat_smile:

Reply:
I think the options are:

* docs.rs
* crates.parity.io
* substrate.dev/rustdocs/alpha.8
* substrate.dev/rustdocs/master

All of them are used at various places in the knowledge base

Reply:
I think `crates.parity.io` is what @joepetrowski prefers since it is pinned to `master`.